ui/src/keyboardLayouts/en_US.ts

```diff
@@ -103,11 +103,18 @@ export const chars = {
   "\n": { key: "Enter", shift: false },
   Enter: { key: "Enter", shift: false },
   Tab: { key: "Tab", shift: false },
-  PrintScreen: { key: "Prt Sc", shift: false },
-  SystemRequest: { key: "Prt Sc", shift: true },
+  PrintScreen: { key: "PrintScreen", shift: false },
```

Comment:
This was an error I introduced (that couldn't have affected anything yet, since we're not exposing these as paste-keys), but the key value must match the [keycode mapping](https://github.com/dlorch/kvm/pull/2/files#diff-e0624f1a2e05f1612a6f9615321679d3bb2eae2990f5d47cae87fad2543e8b64R246) in _keyboardMappings.ts_